ui/src/app/components/cromwell-configuration-panel.tsx

```diff
@@ -53,7 +53,7 @@ const PanelMain = fp.flow(
   }) => {
     // all apps besides Jupyter
     const [userApps, setUserApps] = useState<UserAppEnvironment[]>();
-    const [creationStarted, setCreationStarted] = useState(false);
+    const [creatingStarted, setCreatingStarted] = useState(false);
```

Comment:
creating by itself, not started :) meaning "we are currently creating"

the rest looks good, thanks!

Reply:
Whoops. Thanks.